client/templates/juror-management/_partials/heading.njk

```diff
@@ -31,7 +31,7 @@
         }) }}
       {% endif %}
 
-      {% if not hasSummons and canSummon %}
+      {% if canSummon %}
```

Comment:
The name of this variable might need to be canEnterSummons - just to make it distinguish from can summon jurors to a pool

Reply:
Good point on that :-)

Reply:
updated